widget/ui/src/components/StepDetails/StepDetails.tsx

```diff
@@ -33,8 +35,20 @@ const StepDetailsComponent = forwardRef<HTMLDivElement, StepDetailsProps>(
       tabIndex,
       tooltipContainer,
     } = props;
+
+    const { from, to, swapper } = step;
     const containerRef = useRef<HTMLDivElement>(null);
     const isCompleted = state === 'completed' || state === 'error';
+    const swappers: InternalSwap[] = step.internalSwaps?.length
```

Comment:
NIT: I think `internalSwaps` better communicate this variable purpose.

Reply:
Because the details of each step are used for two places (quote-details and swap-progress) and there are different styles for each of them, I have considered a separate section for each of them. For the internal step, if we want to separate it, the complexity of the code would increase.